doc/prop.xml

```diff
@@ -8,6 +8,83 @@
 #############################################################################
 ##
 
+<#GAPDoc Label="IsPartialOrderDigraph">
+<ManSection>
+  <Prop Name="IsPartialOrderDigraph" Arg="digraph"/>
+  <Returns><K>true</K> or <K>false</K>.</Returns>
+  <Description>
+    This is a synonym for <Ref Prop="IsReflexiveDigraph"/> and
+    <Ref Prop="IsAntisymmetricDigraph"/> and <Ref Prop="IsTransitiveDigraph"/>.
+    We will consider a digraph as a partial order with relation defined by
+    <A>x</A> <M>\leq</M> <A>y</A> if and only if <A>(x,y)</A> is an edge.
+
+    <Example><![CDATA[
+gap> gr := Digraph([[1, 3], [2, 3], [3]]);
+<digraph with 3 vertices, 5 edges>
+gap> IsPartialOrderDigraph(gr);
+true
+gap> gr := CycleDigraph(5);
+<digraph with 5 vertices, 5 edges>
+gap> IsPartialOrderDigraph(gr);
+false
+]]></Example>
+  </Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="IsMeetSemilatticeDigraph">
+<ManSection>
+  <Prop Name="IsMeetSemilatticeDigraph" Arg="digraph"/>
+  <Prop Name="IsJoinSemilatticeDigraph" Arg="digraph"/>
+  <Prop Name="IsLatticeDigraph" Arg="digraph"/>
+  <Returns><K>true</K> or <K>false</K>.</Returns>
+  <Description>
+    
+    <C>IsMeetSemilatticeDigraph</C> returns <K>true</K> if the digraph
+    <A>digraph</A> is a meet semilattice; <C>IsJoinSemilatticeDigraph</C>
+    returns <K>true</K> if the digraph <A>digraph</A> is a join semilattice;
+    and  <C>IsLatticeDigraph</C> returns <K>true</K> if the digraph
+    <A>digraph</A> is both a meet and a join semilattice.
+    <P/>
+
+    We consider a partial order digraph <Ref Prop="IsPartialOrderDigraph"/>
+    to be a digraph with the relation <A>x</A> <M>\leq</M> <A>y</A> if and only
+    if <A>(x,y)</A> is an edge. A digraph is a <E>meet semilattice</E> if it is
+    a partial order and every pair of vertices has a unique greatest lower bound
```

Comment:
The word "unique" should be removed since the word "greatest" implies uniqueness.